api/users/users.model.ts

```diff
@@ -0,0 +1,8 @@
+import { fnSchema, model } from '../../src';
```

Comment:
Where is `fnSchema` exported?

Reply:
fixed, createSchema was used instead of fnSchema